### DISCUSSION TITLE
TFT35 screen compatibly

### DISCUSSION THREAD

**Discussion by Camsinner** [mriscoc/Ender3V2S1#295] [Ideas]
Hi.
So I recently upgraded to the SKR mini E3 3.0 board and didn't want to have to repin to use the old ender 3 v2 screen.
I ended up buying a TFT35 duel mode touch screen to try and use with the new board and your firmware for it.

When I flash your firmware on the board the screen stops working correctly and keeps saying printer not connected.

Would it be possible to get a version of your firmware for this?

**Comment by mriscoc** [comment]
Sorry, this firmware is only compatible with DWIN/DACAI color screens.
